docker/images/pinot/etc/jmx_prometheus_javaagent/configs/broker.yml

```diff
@@ -112,6 +118,12 @@ rules:
 - pattern: "\"org\\.apache\\.pinot\\.common\\.metrics\"<type=\"BrokerMetrics\", name=\"pinot\\.broker\\.routingTableUpdateTime\"><>(\\w+)"
   name: "pinot_broker_routingTableUpdateTime_$1"
   cache: true
+- pattern: "\"org\\.apache\\.pinot\\.common\\.metrics\"<type=\"BrokerMetrics\", name=\"pinot\\.broker\\.adaptiveServerSelectorType\"><>(\\w+)"
+  name: "pinot_broker_adaptiveServerSelectorType_$1"
```

Comment:
Can we please add lables here?

Reply:
Its does not have any extracted info to add as a label. More of a static config info metric